Containers/mastercontainer/Dockerfile

```diff
@@ -52,38 +75,39 @@ RUN set -e && \
     cd ..; \
     rm -f /usr/local/bin/composer; \
     chmod 770 -R ./; \
+    chmod 777 -R /root; \
     chown www-data:www-data -R /var/www; \
     rm -r ./php/data; \
-    rm -r ./php/session
+    rm -r ./php/session; \
+    apk del --no-cache git
 
 RUN mkdir -p /etc/apache2/certs && \
     cd /etc/apache2/certs && \
     openssl req -new -newkey rsa:4096 -days 3650 -nodes -x509 -subj "/C=DE/ST=BE/L=Local/O=Dev/CN=nextcloud.local" -keyout ./ssl.key -out ./ssl.crt;
 
 COPY mastercontainer.conf /etc/apache2/sites-available/
 
-RUN a2enmod rewrite \
-    headers \
-    env \
-    mime \
-    dir \
-    authz_core \
-    proxy \
-    proxy_http \
-    ssl
-
-RUN rm /etc/apache2/ports.conf; \
-    sed -s -i -e "s/Include ports.conf//" /etc/apache2/apache2.conf; \
-    sed -i "/^Listen /d" /etc/apache2/apache2.conf
+RUN sed -i \
+		-e '/^Listen /d' \
+		-e 's/User apache/User www-data/g' \
+		-e 's/Group apache/Group www-data/g' \
+		-e 's/^#\(LoadModule .*mod_rewrite.so\)/\1/' \
+		-e 's/^#\(LoadModule .*mod_headers.so\)/\1/' \
+		-e 's/^#\(LoadModule .*mod_env.so\)/\1/' \
+		-e 's/^#\(LoadModule .*mod_mime.so\)/\1/' \
+		-e 's/^#\(LoadModule .*mod_dir.so\)/\1/' \
+		-e 's/^#\(LoadModule .*mod_authz_core.so\)/\1/' \
+		-e 's/^#\(LoadModule .*mod_mpm_event.so\)/\1/' \
+		-e 's/\(LoadModule .*mod_mpm_worker.so\)/#\1/' \
+		-e 's/\(LoadModule .*mod_mpm_prefork.so\)/#\1/' \
```

Comment:
I think proxy, proxy_http and ssl are missing here?

Reply:
I will have look later

Reply:
done